server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -1327,6 +1327,118 @@
           "to": []
         }
       }
+    },
+    {
+      "allow": {
+        "from":[
+          {
+            "kind": "PriorityClass",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["name"]
+              ],
+              "description": "PriorityClass in Kubernetes assigns importance levels to Pods, influencing scheduling decisions. The name of the PriorityClass is referenced by other k8s components"
```

Comment:
Let's not abbreviate "Kubernetes" or if we do, make sure that we abide by their branding request that "K8s" be capitalized. Having said that, let's not abbreviate Kubernetes. The PriorityClass name is referenced by other components... for what purpose?